sos/plugins/origin.py

```diff
@@ -107,9 +118,16 @@ def setup(self):
         if self.is_master():
             self.add_copy_spec([
                 self.master_cfg,
+                self.master_env,
                 os.path.join(self.master_base_dir, "*.crt"),
             ])
 
+            if self.is_static_pod_compatible():
+                self.add_copy_spec([
+                    os.path.join(self.static_pod_dir, "*.yaml"),
+                ])
```

Comment:
No need to use a list for a single item - `add_copy_spec()` accepts either a string, or a list of strings. The list is only needed if there is > 1 item (and this often helps to avoid the need for line breaks).